tsdb/head_test.go

```diff
@@ -2086,6 +2086,81 @@ func TestWalRepair_DecodingError(t *testing.T) {
 	}
 }
 
+// TestWblRepair_DecodingError ensures that a repair is run for an error
+// when decoding a record.
+func TestWblRepair_DecodingError(t *testing.T) {
+	var enc record.Encoder
+	corrFunc := func(rec []byte) []byte {
+		return rec[:3]
+	}
+	rec := enc.Samples([]record.RefSample{{Ref: 0, T: 99, V: 1}}, []byte{})
+	totalRecs := 9
+	expRecs := 5
+	t.Run("decoding samples", func(t *testing.T) {
```

Comment:
Nit: since there's only one test case in the TestWblRepair_DecodingError function I don't think we need to make a subtest with t.Run()

Reply:
Thanks addressed :+1: